### PR TITLE
[plugin.video.maldivestv@matrix] 1.1.0

### PR DESCRIPTION
**Pull request by jawish**
### Add-on details:

- **General**
  - Add-on name: Maldives TV
  - Add-on ID: plugin.video.maldivestv
  - Version number: 1.1.0
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/jawish/plugin.video.maldivestv
  
Kodi video plugin to view live streams of Maldives TV channels.

### Description of changes:

Initial release!

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.maldivestv/addon.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+<addon id="plugin.video.maldivestv" version="1.1.0" name="Maldives TV" provider-name="Jawish Hameed">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+		<import addon="script.module.requests"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="addon.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en">Maldives TV</summary>
```
Comment (romanvm) [inline plugin.video.maldivestv/addon.xml:11]: Please use 4-letter language codes: ll_CC, e.g. en_GB

File: plugin.video.maldivestv/addon.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <description lang="en">Kodi video plugin to view live streams of Maldives TV channels.</description>
+    <disclaimer lang="en">Live streams are currently from Dhiraagu play.mv.</disclaimer>
+    <platform>all</platform>
+    <license>GNU General Public License v2.0</license>
```
Comment (romanvm) [inline plugin.video.maldivestv/addon.xml:15]: Please use a proper license code: https://spdx.org/licenses/

**Comment by romanvm** [comment]
Please fix fanart size.

**Review by romanvm — CHANGES_REQUESTED** [review]

**Comment by basrieter** [comment]
ping @jawish

**Pull request closed without merging**